internal/monocular/doc.go

```diff
@@ -15,7 +15,7 @@ limitations under the License.
 */
 
 // Package monocular contains the logic for interacting with monocular instances
-// like the Helm Hub.
+// like the original version of Helm Hub.
```

Comment:
Thinking about this, does it make more sense as follows:
"like the Artifact Hub." ?

Reply:
Helm Hub v1 used monocular. Artifact Hub does not, as far as I can tell?

Reply:
Ok, if that is so maybe remove it as Helm Hub no longer exists?